src/test/java/seedu/address/logic/commands/util/CommandListTest.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.logic.commands.util;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+
+class CommandListTest {
+
+    @Test
+    void getCommandToUsageMapping() {
+        HashMap<String, String> commands = CommandList.getCommandToUsageMapping();
+
+        assertTrue(commands.containsKey("list"));
+        assertTrue(commands.containsKey("log"));
+
+        assertFalse(commands.containsKey("unknown"));
+    }
+}
```

Comment:
Right now the CommandList constructor is not being called, hence it says it is not covered in code coverage.

Perhaps you can consider this link if you think it is necessary, otherwise I think it is fine to leave it as red as well, just a numbers game

https://www.nerd.vision/post/jacoco-coverage-of-util-classes

Reply:
Thanks for sharing this link!